ansible/master/ansible-jenkins/tasks/cli.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Handle plugins
+- name: "{{ startup_delay_s | default(10) }}s delay while starting Jenkins"
+  wait_for: host=localhost port={{ port }} delay={{ startup_delay_s | default(10) }}
```

Comment:
nitpick: I prefer the args to ansible modules to be one on a line in "key: value" format.  I find it easier to read and makes diffs nicer.